firmware_motion_board/src/main.c

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+  Main Source File
+
+  Company:
+    Microchip Technology Inc.
+
+  File Name:
+    main.c
+
+  Summary:
+    This file contains the "main" function for a project.
+
+  Description:
+    This file contains the "main" function for a project.  The
+    "main" function calls the "SYS_Initialize" function to initialize the state
+    machines of all modules in the system
+ *******************************************************************************/
+
+// *****************************************************************************
+// *****************************************************************************
+// Section: Included Files
+// *****************************************************************************
+// *****************************************************************************
+
+#include <stddef.h>                     // Defines NULL
+#include <stdbool.h>                    // Defines true
+#include <stdlib.h>                     // Defines EXIT_FAILURE
+#include "definitions.h"                // SYS function prototypes
+
+#include <stdio.h>
+#include <stdint.h>
+
+#include "motor.h"
+#include "control_system.h"
+#include "protocol.h"
+// *****************************************************************************
+// *****************************************************************************
+// Section: Main Entry Point
+// *****************************************************************************
+// *****************************************************************************
+
+int fputc(int ch, FILE *f)
+{
+    while (UART3_WriteIsBusy());
+    UART3_Write(&ch, 1);
+
+    return 0;
+}
+
+int32_t prev_left = 0, prev_right = 0;
+
+int main(void)
+{
+    /* Initialize all modules */
+    SYS_Initialize(NULL);
+    CORETIMER_Start();
+
+    QEI3_Start();   // odometry left
+    QEI1_Start();   // odometry right
+    
+    motor_init();   
+    control_system_initRegulators();

```

Comment:
```suggestion
    control_system_init();

```
Ja bih da se mi drzip underscore konvencije

Reply:
Underscore, reseno.